src/App.tsx

```diff
@@ -17,18 +60,27 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter values={todoFiler} set={setTodoFillter} />
```

Comment:
Typographical error: The prop `set` should be `setTodoFilter` to match the corrected state setter name.